docs/index.rst

```diff
@@ -73,32 +76,22 @@ all of the cells in the notebook. In each will be a graphical interface to enter
 When the photometry is done there will be a new notebook called `photometry_run.ipynb` that will have a record
 of the photometry that was done.
 
-
-Command line interface for generating settings
-----------------------------------------------
-
-To generate settings using the command line, run the following command::
-
-    stellarphot-settings
-
-
-This will generate a settings file in the directory in which you run the command
-called `stellarphot_settings.json`. Edit that file in the editor of your choice.
-
 Editing a settings file directly
 --------------------------------
 
 The settings file is a JSON file that can be edited in any text editor.
 
 
-Performing photometry
----------------------
+Performing photometry from the command line
+-------------------------------------------
```

Comment:
I don't like `from the command line` here since this is really just in a Python script... try
```suggestion
Performing photometry from within a Python script
-------------------------------------------------
```